docs/README-agent-install.md

```diff
@@ -35,15 +45,22 @@ Puppet agent software.
 
 #### Environment
 
-NX-OS supports two possible environments for running third party software:
-`bash-shell` and `guestshell`. Choose one environment for running the
-Puppet agent software. You may run Puppet from either environment but not from both
-at the same time.
+NX-OS supports three possible environments for running third party software:
+`bash-shell`, `guestshell` and the `open agent container (OAC)`.
+
+Environment                  | Supported Platforms                      |
+-----------------------------|------------------------------------------|
+`bash-shell` or `guestshell` | Cisco Nexus 30xx, 31xx, 93xx, 95xx, N9Kv |
+`open agent container`       | Cisco Nexus 56xx, 60xx, 7xxx             |
```

Comment:
I would add `(OAC)` after each oac reference since I think most people will refer to it by the abbreviation, ie:

`open agent container (OAC)`